src/ralph/api/auth/oidc.py

```diff
@@ -109,17 +108,25 @@ def get_oidc_user(
     Raises:
         HTTPException
     """
+
     if auth_header is None or "Bearer" not in auth_header:
-        logger.error("The OpenID Connect authentication mode requires a Bearer token")
-        raise HTTPException(
-            status_code=status.HTTP_401_UNAUTHORIZED,
-            detail="Could not validate credentials",
-            headers={"WWW-Authenticate": "Bearer"},
+        logger.info(
+            "Not using OIDC auth. The OpenID Connect authentication mode requires a "
+            "Bearer token"
```

Comment:
IMO this is for debug purpose. This would be too noisy and uninformative in production.

Reply:
agreed. changed.